backend/main.py

```diff
@@ -66,6 +67,12 @@ def ValidateInputs(present, valid):
     elif a == 'missionId':
       if not firebase.get('/missions/%s/name' % data, None):
         raise InvalidInputError('Mission %s not found.' % data)
+    elif a == 'chatRoomId':
+      if not firebase.get('/chatRooms/%s/name' % data, None):
+        raise InvalidInputError('Chat room %s not found.' % data)
+    elif a == 'allegianceFilter':
+      if data not in ('horde', 'resistance', 'none'):
```

Comment:
We will want to make it easy to add more valid allegiances over time. Could we put the list of valid options into constants? Or do you think this will be the only place where we will have to update it in the future?